### DISCUSSION TITLE
Could you please explain how to implement a proxy chain?

### DISCUSSION THREAD

**Discussion by Luban-Yuumi** [hyperium/hyper#3452] [Q&A]
I have read the examples/http_proxy.rs. Can you advise on how to forward the request to another HTTP or HTTPS or Socks5 proxy?

**Comment by seanmonstar** [comment]
I would normally have said to look at that example. I'm not sure what part you need more advice on. Care to add more to your question?

**Reply by Luban-Yuumi** [reply]
Provide a local HTTP proxy to forward traffic to a remote HTTP, HTTPS, or SOCKS5 proxy

**Reply by seanmonstar** [reply]
That's the same thing you asked before. You wish to program a local proxy? You can take a look at that example, and adapt it to what you need. I can't write the program for you.

**Reply by Luban-Yuumi** [reply]
Thank you very much for your reply, I'll do some research on my own.
